newsletter-bundle/src/Resources/contao/classes/Newsletter.php

```diff
@@ -932,9 +932,13 @@ public function updateAccount()
 
 	/**
 	 * Purge subscriptions that have not been activated within 24 hours
+	 *
+	 * @deprecated Deprecated since Contao 5.0, to be removed in Contao 6.0.
 	 */
 	public function purgeSubscriptions()
 	{
+		trigger_deprecation('contao/newsletter-bundle', '5.0', 'Calling "%s::%s()" has been deprecated and will no longer work in Contao 6.0.', __CLASS__, __METHOD__);
```

Comment:
```suggestion
		trigger_deprecation('contao/newsletter-bundle', '5.0', 'Calling "%s()" has been deprecated and will no longer work in Contao 6.0.', __METHOD__);
```

Reply:
Good catch, there are actually a few places in the core where `__CLASS__` and `__METHOD__` is used in the deprecation message like this.

Reply:
Yes, I used it like that in the past I think 🙈

Reply:
https://github.com/contao/contao/pull/4643 🙃